tests/test_cart_model.py

```
@@ -0,0 +1,144 @@
+
+        assert repr(cart) == f"<Cart(user_id={user.id})>"
+
+    def test_cart_unique_user(self, session, user):
+        cart1 = Cart(user_id=user.id)
+        session.add(cart1)
+        session.commit()
+
+        cart2 = Cart(user_id=user.id)
+        session.add(cart2)
+
+        with pytest.raises(Exception):
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)